examples/knative_postgresql_customvar/README.md

```diff
@@ -182,28 +182,28 @@ After the application is built we can check the `Services` under `Serverless` vi
 
 Now, the only thing that remains is to connect the DB and the application. We let the Service Binding Operator to make the connection for us.
 
-Create the following `ServiceBindingRequest`:
+Create the following `ServiceBinding`:
 
 ```shell
 cat <<EOS |kubectl apply -f -
 ---
-apiVersion: apps.openshift.io/v1alpha1
-kind: ServiceBindingRequest
+apiVersion: operators.coreos.com/v1alpha1
+kind: ServiceBinding
 metadata:
   name: binding-request
   namespace: service-binding-demo
 spec:
-  applicationSelector:
+  application:
     group: serving.knative.dev
     version: v1beta1
     resource: services
-    resourceRef: knative-app
-  backingServiceSelector:
-    group: postgresql.baiju.dev
-    version: v1alpha1
-    kind: Database
-    resourceRef: db-demo
-  customEnvVar:
+    name: knative-app
+  - services:
```

Comment:
Same [comment](https://github.com/redhat-developer/service-binding-operator/pull/495/files#r431806733)

Reply:
Some of the other example files have the same issue

Reply:
fixed https://github.com/redhat-developer/service-binding-operator/pull/495/commits/5fc61873c3a020c60410180ef70a435d4381f744 thanks!